includes/reader-revenue/my-account/class-woocommerce-my-account.php

```diff
@@ -896,6 +897,25 @@ public static function handle_email_change_request( $user_id ) {
 		exit;
 	}
 
+	/**
+	 * Handle admin email change request.
+	 *
+	 * @param int     $user_id User ID.
+	 * @param WP_User $user    User object.
+	 * @param array   $data    User data.
+	 */
+	public static function handle_admin_email_change_request( $user_id, $user, $data ) {
+		if ( ! is_admin() || ! self::is_email_change_enabled() ) {
+			return;
+		}
```

Comment:
Should this check be preventing an email update? Bailing here just means the email change happened without the required syncs.

Reply:
I believe so. We bail here if the request is not from admin since non-admin requests are handled via a different hook, or if the feature is not enabled to maintain the existing behavior.

Reply:
Oops, just realized I didn't directly answer your first question:

> Should this check be preventing an email update?

No. At least we aren't doing this currently, so my assumption is this is expected.

Reply:
Thanks for explaining! I believe we could go without this clause. `is_email_change_enabled()` is not whether the sync should happen, it's whether we allow a reader to change their email. Non-blocking, though!